api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala

```diff
@@ -513,8 +500,7 @@ class ApiWorksTest
                           |     "title": "${work1.title}",
                           |     "creators": [],
                           |     "subjects": [ ],
-                          |     "genres": [ ],
-                          |     "items": [ ]
+                          |     "genres": [ ]
```

Comment:
This test is about rendering items with no ids, but items have been taken out? Or am I missing something?

Reply:
No, that’s on L194 and 210 – this test is

> should return matching results if doing a full-text search